codespell_lib/data/dictionary.txt

```diff
@@ -18264,6 +18264,7 @@ jumo->jump
 jumoed->jumped
 jumpimng->jumping
 jumpt->jumped, jump,
+juipter->Jupiter
```

Comment:
This is not in sorted order. Please move it to after `"juducial->judicial"`.

```suggestion
```

Reply:
Yeah that's sort of the point, see https://github.com/codespell-project/codespell/pull/2251#issuecomment-1020349080 and https://github.com/codespell-project/codespell/pull/2251#issuecomment-1022709986